apps/src/code-studio/videos.js

```diff
@@ -226,36 +226,35 @@ videos.showVideoDialog = function(options, forceShowVideo) {
     'text-align': 'right'
   });
   nav.append(fallbackPlayerLinkDiv);
+  function onResize() {
```

Comment:
This is the first function to be indented (and scoped as such)... does it need to be?

Reply:
Do you mean "inlined" or are you referring to the indentation?